BaseApp/COD3009.TXT

```diff
@@ -32,6 +32,13 @@ OBJECT Codeunit 3009 DotNet_MemoryStream
       DotNetMemoryStream := DotNetMemoryStream.MemoryStream(DotNetArray)
     END;
 
+    [External]
```

Comment:
Why do we need this function? The same can be achieved by first calling InitMemoryStream and then CopyToInStream. We should try to avoid adding helper functions into wrappers. Maybe create a separate type helper?